inc/cloner/class-cloner.php

```diff
@@ -1561,7 +1561,7 @@ protected function cloneSection( $section_id, $post_type, $parent_id = null ) {
 		$content = $this->retrieveH5P( $content );
 
 		// Set title and content
-		$section['title'] = $section['title']['rendered'];
+		$section['title'] = $section['title']['raw'] ?? $section['title']['rendered'];
```

Comment:
Use the `raw` title instead of the `rendered` when available -- this removes the preceding "Private: " on the title of the section.